src/Graphql/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -65,6 +68,12 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             }
 
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
+            if (null !== $this->resourceAccessChecker) {
```

Comment:
Shouldn't you put this in an abstract or a service? This is duplicated in `CollectionResolverFactory`.

Reply:
And in `ItemResolver`.

Reply:
I've moved this logic in a trait in the meantime.